### PR TITLE
turn on debug logging in KBFS on mobile

### PR DESCRIPTION
**Pull request by maxtaco**
remove old comments

### CODE REVIEW

**Comment by maxtaco** [comment]
As @strib suggests, it wasn't a KBFS bug, but a `gobind` bug.  Not tested....

File: go/bind/keybase.go [hunk 1]
```diff
@@ -86,10 +86,11 @@ func Init(homeDir string, logFile string, runModeStr string, accessGroupOverride
...
 	}
 
-	// FIXME (MBG): This is causing RPC responses to sometimes not be recieved
-	// on iOS. Repro by hooking up getExtendedStatus to a button in the iOS
-	// client and watching JS logs. Disabling until we have a root cause / fix.
 	kbfsParams := libkbfs.DefaultInitParams(kbCtx)
+
+	// Setting this flag will enable KBFS debug logging to alway be
+	// true in a mobile setting. Kill this setting if too spammy.
+	kbfsParams.Debug = true
```
Comment (mmaxim) [inline go/bind/keybase.go]: I made a mistake not putting these parameters settings in the goroutine, you think we could add them in this PR?

**Comment by maxtaco** [comment]
ok, tried it with this commit cbb5153

**Comment by maxtaco** [comment]
Merging despite CI failures, which I believe are unrelated.

**Pull request merged**